test/unit/sql/blockRewards.js

```diff
@@ -77,17 +77,12 @@ describe('BlockRewardsSQL', function () {
 		dbSandbox = new DBSandbox(modulesLoader.scope.config.db, 'lisk_test_sql_block_rewards');
 		dbSandbox.create(function (err, __db) {
 			db = __db;
-
 			// Force rewards start at 150-th block
```

Comment:
Wtf? Not remember that we started rewards on 150-th block on mainnet.

Reply:
I am not sure I understand what this is doing, since i dont see `constants.rewards.offset` being changed anywhere else before this. We are just setting the variables to the same as they are in constants anyways...

Reply:
It probably because for testnet we use different data.